REST_API_DOCS.md

```
@@ -0,0 +1,1216 @@
+
+| 헤더 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| Authorization | string | O | Bearer |
+
+**Path Parameters**
+
+| 파라미터 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| postId | integer | O | 삭제할 게시글의 ID |
+
+**Request Example**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)